docs/lineage/lineage-feature-guide.md

```diff
@@ -0,0 +1,129 @@
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
+
+# About DataHub Lineage
+
+<FeatureAvailability/>
+
+In DataHub, lineage is how describe the way that data flows within and between your source systems. For a given entity, lineage allows you to see where this data is coming from (what's upstream) and where it's going (what's downstream). For more information, see [this video](https://www.youtube.com/watch?v=rONGpsndzRw&ab_channel=DataHub) for Lineage 101 in DataHub.
+
+Starting in version `0.9.5`, DataHub supports the manual editing of lineage between entities. Data experts are free to add or remove upstream and downstream lineage edges in both the Lineage Visualization screen as well as the Lineage tab on entity pages. Use this feature to supplement automatic lineage extraction in ingestion or establish important entity relationships in sources that we don't support automatic extraction for yet! Editing lineage by hand is only supported for Datasets, Charts, Dashboards, and Data Jobs for now.
+
+:::note
+
+Lineage added by hand and programmatically may conflict with one another to cause unwanted overwrites. It is strongly recommend that lineage is edited manually in cases where lineage information is not also extracted in automated fashion, e.g. by running an ingestion source.
+
+:::
+
+## Manual Lineage Setup, Prerequisites, and Permissions
```

Comment:
At some point we'll also want to generalize this. Maybe we just call this "Lineage Setup, Prerequisite..."

Reply:
yup! i'll call it that here